### ISSUE TITLE
Bert - pick() in language_models.py can return no candidates

### ISSUE THREAD

**Issue by chiragjn** [makcedward/nlpaug#68]
For bert, the `get_candidiates` in `pick` function
```
    def pick(self, logits, target_word, n=1):
        candidate_ids, candidate_probas = self.prob_multinomial(logits, n=n*10)
        results = self.get_candidiates(candidate_ids, candidate_probas, target_word, n)

        return results
```
can return no candidates when all tokens are sub tokens (starts with '##')

Should pick return a non-skip token greedily by looking at the logits when `results`  is empty?

**Comment by makcedward** [comment]
There is a [retry mechanism](https://github.com/makcedward/nlpaug/blob/master/nlpaug/augmenter/word/context_word_embs.py#L224) as a remedy solution.

**Comment by chiragjn** [comment]
Oh, that's good to know, looking at the code, I think this is missing from the `.insert` augmentation which caused the crash for me

**Comment by makcedward** [comment]
Oh, you are right. It missed in insert function. Will add it
